scene/gui/graph_edit.cpp

```diff
@@ -1335,7 +1334,6 @@ GraphEdit::GraphEdit() {
 	top_layer->set_mouse_filter(MOUSE_FILTER_PASS);
 	top_layer->set_anchors_and_margins_preset(Control::PRESET_WIDE);
 	top_layer->connect("draw", this, "_top_layer_draw");
-	top_layer->set_mouse_filter(MOUSE_FILTER_PASS);
```

Comment:
This one is not a Container, but the mouse filter was already set to `Pass` on line 1335.